modules/hub/pages/manual-mode.adoc

```diff
@@ -0,0 +1,56 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+In this example, manual mode is used, where user needs to provide credentials for each {productname} server user wants to connect to.
+
+Here in this example, following is happening
+
+[square]
+* User provide credentials for the hub to the login method, a `sessionkey` for the hub is returned
+* Using the sessionkey from the previous step, `serverIds` for the servers attached to the hub are requested
+* User provide the credentials for each {productname} server, she needs data from and calls the hub method `attachToServers`. Authentication part is done here.
+* `multicast` call is being performed on the specific servers where `serverId` is used to identify the specific sever. Behind the scenes, `system.list_system method` of each {productname} server's xmlrpc api method will be called to get the list of the clients registed with each {productname} server
+* Hub will take care of collecting these results and return in form of a `map`. This map has two entries, entry with `Successfull`  key, has list of responses where call was successful while with `Failed` key  has the failed ones.
```

Comment:
I would reword it to use 'aggregating' instead of 'collecting'.

The Hub will aggregate the results and return a response in form of a `map`, which has two entries:
`Successful`: contains a list of responses for those clients where the call was successful.
`Failed`: contains a list of responses  for those clients where the call failed.